### PR TITLE
php-cs-fixer 3.0.0

### PR DESCRIPTION
**Pull request by derrabus**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/php-cs-fixer.rb [hunk 1]
```diff
@@ -20,7 +20,9 @@ def install
       <?php $this->foo(   'homebrew rox'   );
     EOS
     (testpath/"correct_test.php").write <<~EOS
-      <?php $this->foo('homebrew rox');
+      <?php
+      
+      $this->foo('homebrew rox');
```
Comment (carlocab) [inline Formula/php-cs-fixer.rb:25]:
```suggestion
      <?php

      $this->foo('homebrew rox');
```

File: Formula/php-cs-fixer.rb [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+  url "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v3.0.0/php-cs-fixer.phar"
+  sha256 "993f3d300db32c4158891bc8a13a7f91f7dac85ce4eda209b3582f0e329b3990"
   license "MIT"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "d1336616dc3319ea53d1a6e299b879fbee7df68c0bbe8ce4c14a01ba33b53791"
   end
 
-  depends_on "php" if MacOS.version <= :el_capitan
+  depends_on "php" if MacOS.version <= :high_sierra
```
Comment (carlocab) [inline Formula/php-cs-fixer.rb:12]:
```suggestion
  uses_from_macos "php", since: :catalina
```

Comment (derrabus) [reply Formula/php-cs-fixer.rb:12]: PHP CS Fixer 3.0.0 requires PHP 7.1 or newer. IIRC, Mojave bundled PHP 7.1 already.

Comment (carlocab) [reply Formula/php-cs-fixer.rb:12]: Oops, definitely meant to type Mojave

**Review by carlocab — APPROVED** [review]
Thanks @derrabus!

**Comment by boboldehampsink** [comment]
This is a breaking change. Would love to see `php-cs-fixer@2` available as well

**Comment by derrabus** [comment]
Feel free to contribute such a formula, @boboldehampsink. 🙂

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew

**Comment by boboldehampsink** [comment]
Here is my contribution for a `php-cs-fixer@2`: https://github.com/Homebrew/homebrew-core/pull/77019

**Pull request closed without merging**